app/components/common/measureCommitteeList/measureCommitteeList.html

```diff
@@ -1,6 +1,6 @@
 <ul>
   <li ng-repeat="committee in committees">
-    <h4><a ui-sref="appMain.committee.main({committee_id: committee.committee_id})">{{ committee.name }}</a></h4>
+    <h4><a ui-sref="appMain.committee.main({filer_id: committee.filer_id})">{{ committee.name }}</a></h4>
```

Comment:
Yeah, I think this changed while I was working on it. At one point most objects had an id and a filter_id. Since those now seem to be merged, we should change this.